opendbc/car/toyota/carcontroller.py

```diff
@@ -42,7 +42,7 @@ def get_long_tune(CP, params):
   kdBP = [0.]
   kdV = [0.]
   if CP.carFingerprint in TSS2_CAR:
-    kiV = [0.5]
+    kiV = [0.25]
```

Comment:
We can bump this back up to 0.5 once we figure out a good way to winddown integral for step changes using error rate). See this case where it shouldn't have wound down integral:

6483a59cd49b6650/000001cb--9815f176ae/1

![image](https://github.com/user-attachments/assets/cd648fca-7590-4b93-9418-f114608d36ab)